### PR TITLE
Tf

### PR DESCRIPTION
**Pull request by ghosalsattam**
This Pull request is to create a basic "gaussian kernel op". It is necessary image processing tool and is often used with other methods like CannyEdgeDetection etcc to reduce noise etc. So it is an important image processing tool.

### CODE REVIEW

**Comment by ghosalsattam** [comment]
@rthadur @tensorflow-jenkins @tensorflower-gardener @tanzhenyu Please provide a review.

File: .github/workflows/pythonpackage.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
+name: Python package
```
Comment (mihaimaruseac) [inline .github/workflows/pythonpackage.yml]: This file should not exist

File: tensorflow/python/keras/layers/convolutional.py [hunk 2]
```diff
@@ -3030,6 +5622,15 @@ def get_config(self):
 SeparableConvolution1D = SeparableConv1D
 SeparableConvolution2D = SeparableConv2D
 Convolution2DTranspose = Conv2DTranspose
+<<<<<<< HEAD
```
Comment (mihaimaruseac) [inline tensorflow/python/keras/layers/convolutional.py]: Merge conflict markers. These should not be left in the PR

**Review by mihaimaruseac — CHANGES_REQUESTED** [review]
Does this really need to be an addition to TF? Can it be made to TensorFlow Addons instead?

Also, the PR should have tests (https://www.tensorflow.org/community/contribute/tests). Also, read documentation on how to contribute to the code (https://www.tensorflow.org/community/contribute/tests) and the coding style (https://www.tensorflow.org/community/contribute/code_style).

Finally, the additions should not be in the root of the package.

**Comment by tanzhenyu** [comment]
+1 on making this part of Addons, not TF core.

**Comment by tanzhenyu** [comment]
Or actually keras-image as a separate repo for image related things, for what it's worth

**Comment by ghosalsattam** [comment]
I have added the file to keras addons

**Comment by mihaimaruseac** [comment]
Please change PR title to something relevant. Please move implementation files to obey the layout of the rest of the code.

Please make sure you follow the links mentioned in the previous review and also consider whether this should be a part of TensorFlow Addons instead.

**Comment by gbaned** [comment]
@ghosalsattam Can you please check @mihaimaruseac's comments and keep us posted. Thanks!

**Comment by aiswaryasankar** [comment]
**🔒 Entelligence AI Vulnerability Scanner**

✅ **No security vulnerabilities found!**

Your code passed our comprehensive security analysis.

📊 **Files Analyzed**: 3 files

---

**Comment by mihaimaruseac** [comment]
This is spam, @aiswaryasankar . Please stop or you will be banned

**Pull request closed without merging**